packages/react/documentation/coding-conventions.md

```diff
@@ -0,0 +1,31 @@
+# Coding Conventions
+
+## Barrel files
+
+In our React package we use barrel files (`index.ts` in the root and in every component folder) to simplify the import of React components.
+Barrel files allow a consumer to do this:
```

Comment:
While ‘consumer’ is an explicit and accurate term, would we be equally clear if we use ‘application’ everywhere and mention ourselves being a ‘library’ as an opposition where useful?

Reply:
I understand. The ambiguousness of ‘consumer’ referring to either an application or its developer seems a bit helpful here.

I still think the term is a bit difficult, but let’s postpone for now.